### PR TITLE
Update Fastly documentation

### PR DESCRIPTION
**Pull request by mattcone**
Hello from Fastly! We're preparing to make our new [customer-facing application](https://www.fastly.com/blog/introducing-next-generation-fastly-control-panel) the default, and we wanted to update your docs to match. The switchover will occur on November 17, so it'd be great if this pull request could be merged on or around that date.

Please let me know if you'd like me to make any changes! Thanks!

### CODE REVIEW

**Comment by eschallock** [comment]
@mattcone Thanks for the pull request! We appreciate the contribution and your effort to keep our docs in sync.

The changes look good to me - I'll make sure that it's merged in on 11/17.

We have started requiring contributors to our Docs and open source libraries to complete a Contributor License Agreement to help clarify the rights of our contributors. Would you be able to complete one of these? You can find find details and a link to the CLA [here](https://github.com/sendgrid/docs#clas-and-cclas).

I hope this isn't too much trouble. Thank you!

File: source/Classroom/Build/Add_Content/content_delivery_networks.md [hunk 1]
```diff
@@ -46,31 +46,31 @@ Once all of this is done, you will need to contact [SendGrid support](https://su
...
+Click the CREATE button. The new service appears in the list of services available.
 
-After provisioning, update the CNAME record for your domain(s) over to the TLS endpoint provided by Fastly support.
+Fastly has [several options](https://docs.fastly.com/guides/securing-communications/ordering-a-paid-tls-option) for SSL termination. If you want to be able to use your SendGrid Email Links WhiteLabel Domain with SSL, you'll need to select either the Shared Certificate, Shared Wildcard Certificate, or Customer Certificate Hosting options. If you need to add your SendGrid whitelabel domain to your Fastly managed certificate, you can open a support ticket with Fastly by emailing support@fastly.com, and they will walk you through the process. Please put "SSL Certificate Request" in the subject and include your [customer ID](https://docs.fastly.com/guides/account-management-and-security/finding-and-managing-your-account-info#finding-your-customer-id).
```
Comment (eschallock) [inline source/Classroom/Build/Add_Content/content_delivery_networks.md]: Could "SendGrid Email Links Whitelabel Domain" simply be "SendGrid link whitelabel domain"?

**Comment by mattcone** [comment]
@eschallock: Thanks for the feedback! I've made the changes you've suggested.

I think Fastly's legal department will need to review the CLA. Would the absence of a completed CLA prevent you from merging this?

**Comment by eschallock** [comment]
@mattcone Thanks for accepting the edits!

Unfortunately, the absence of a CLA would prevent me from merging this PR. I don't have any flexibility regarding that requirement. I apologize for the inconvenience!

Let me know if you have any other questions, and I'll do whatever I can.

**Comment by mattcone** [comment]
eschallock: Thanks for the update. I can't sign the CLA without approval from our legal department. I've asked them to take a look. I'll let you know when I hear back.

**Comment by mattcone** [comment]
@eschallock: I've heard back from our legal team. 

Fastly's general counsel won’t approve the contributor agreement in this context because it is talking about our intellectual property and not yours. We’ll leave this pull request here in case you can make an exception or raise the issue with your legal team.

In the meantime, we’ll file a issue, because your documentation will remain incorrect without the change.

Thanks again for your help.

**Comment by mbernier** [comment]
@mattcone I believe that the reason it mentioned intellectual property is within the context of the shared changes, not Fastly's business. This is because to share the changes between out parties and the public, we need the IP of the contribution to be released.

Having said that, your counsel is not the first and will not be the last to not want to sign the CLA. Their stance is completely reasonable and understandable.

I am wondering, however, if it would make sense to migrate this information from our documentation to yours. It would constitute less upkeep on your part and remove the need for upkeep on our part, because our docs team will never be as knowledgeable about your product as you are. The SendGrid specific part should be relatively static, meaning that our changes to your docs would be less frequent than the other way around.

What do you think?

**Comment by mattcone** [comment]
@mbernier: Thanks for weighing in and suggesting an alternative. 

> I am wondering, however, if it would make sense to migrate this information from our documentation to yours.

I discussed this with my colleagues, and we agreed that this would not be in either party's best interest. One concern is that we don't have any control over the way SendGrid advises their customers to configure their Fastly services, and we don't want to provide recommendations on SendGrid's behalf.

As another alternative, you could revise this pull request to remove the screenshots of the Fastly interface and provide only the essential information (server address and domain name, for example) that your customers need to set up a service in Fastly. That will minimize the changes you'll need to make in the future. You could link to our instructions here:

https://docs.fastly.com/guides/basic-setup/working-with-services#creating-a-new-service

I hope this helps. Please let me know if I can do anything else.

**Comment by mbernier** [comment]
Yeah, that sounds like a great idea. Thanks @mattcone!! Say hey to Gaylin for me!

Hey @eschallock, can you add this last suggestion to the list?

**Comment by eschallock** [comment]
@mbernier @mattcone Yup, this sounds like a good plan to me, I'll let you both know once it's all set.

Thanks!

**Comment by eschallock** [comment]
@mattcone @mbernier I've simplified and duplicated the updated instructions here: https://github.com/sendgrid/docs/pull/2323 I'll close this PR and will merge in #2323 if there are no objections.

Thanks!

**Comment by mbernier** [comment]
Awesome, thanks @eschallock!!

**Pull request closed without merging**